src/main/java/io/codelex/advancedTest/Exercise2/Basket.java

```diff
@@ -0,0 +1,24 @@
+package io.codelex.advancedTest.Exercise2;
+
+public class Basket<T> {
```

Comment:
Te bišķi tāds hibrīds sanācis. :D It kā izmanto generics T, bet tajā pašā laikā masīvs ir Object.. Mans ieteikums tomēr būtu pārtaisīt to Object[] par List<T> un tad tās kļūdas mest atkarībā no counter vērtības